tests/test_core.py

```diff
@@ -265,7 +266,7 @@ def test_get_multiple_build_config_by_labels(self, openshift):  # noqa
         ([http_client.CONFLICT, http_client.OK], False),
         ([http_client.CONFLICT, http_client.CONFLICT, http_client.UNAUTHORIZED], True),
         ([http_client.UNAUTHORIZED], True),
-        ([http_client.CONFLICT for _ in range(10)], True),
+        ([http_client.CONFLICT for _ in range(OS_CONFLICT_MAX_RETRIES + 1)], True),
```

Comment:
Is there a test case where it fails after all attempts are depleted?

Reply:
that test is in test_utils

and I'll make 1 more test which will conflict and success after